### PR TITLE
Katespade

### PR DESCRIPTION
**Pull request by LadyLatitude**
Closes issue #1333.

### CODE REVIEW

**Comment by LadyLatitude** [comment]
Closes issue #1326

File: locations/spiders/motel6.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```
Comment (iandees) [inline locations/spiders/motel6.py]: Can you remove the Motel 6 spider from this PR for Kate Spade please?

File: locations/spiders/katespade.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+                opening_hours = self.parse_hours(store["relationships"]["hours"])
+                if opening_hours:
+                    properties['opening_hours'] = opening_hours
+
+                page = int(re.search(r'page%22:(\d+)', response.url).groups()[0])
+                url = response.urljoin(
+                    response.url.replace("page%22:{}".format(page), "page%22:{}".format(page + 1)))
+
+                yield scrapy.Request(url, headers=HEADERS)
```
Comment (iandees) [inline locations/spiders/katespade.py]: Can this be adjusted so that we only yield one request for the next page instead of one for every store?

**Comment by LadyLatitude** [comment]
Made requested changes and moved to PR #1345

**Pull request closed without merging**